docs/plugins/automelt.rst

```diff
@@ -6,13 +6,41 @@ automelt
     :tags: untested fort productivity items stockpiles
     :no-command:
 
-When `enabled <enable>`, this plugin adds an option to the :kbd:`q` menu for
-stockpiles. When the ``automelt`` option is selected for the stockpile, any
-items placed in the stockpile will automatically be designated to be melted.
+Automelt checks monitor-enabled stockpiles once every in-game day, and will mark valid items for melting.
+
+Please see `gui/automelt` for the interactive configuration dialog.
 
 Usage
 -----
 
 ::
 
     enable automelt
+    automelt [status]
+    automelt (designate)
+    automelt (monitor|nomonitor) <stockpile>
+
+Examples
+--------
+
+Automatically designate all meltable items in the stockpile ("melt") for melting. ::
+
+    enable automelt
+    automelt monitor melt
+
+Enable monitoring for the stockpile ("melt"), and mmediately designate all meltable items in monitored stockpiles for melting. ::
+
+    automelt monitor melt
+    automelt designate
+
+Commands
+--------
+
+``status``
+    Shows current configuration and relevant statistics
+
+``designate``
+    Designates items in monitored stockpiles for melting right now. This works even if ``automelt`` is not currently enabled.
+
+``(no)monitor <stockpile>``
+    Enable/disable monitoring of a given stockpile. Requires the stockpile to have a name set.
```

Comment:
why must it have a name set? why can't we also handle IDs here?

Reply:
Fixed in latest changes. It would already handle enabling by ID, but the GUI didn't show the ID as an alternative to the name if the name was not set.